scheme-libs/racket/unison/sandbox.rkt

```diff
@@ -0,0 +1,149 @@
+(sandbox-builtin "IO.setCurrentDirectory.impl.v3")
+(sandbox-builtin "IO.fileExists.impl.v3")
+(sandbox-builtin "IO.getEnv.impl.v1")
+(sandbox-builtin "IO.getArgs.impl.v1")
+(sandbox-builtin "IO.isDirectory.impl.v3")
+(sandbox-builtin "IO.createDirectory.impl.v3")
+(sandbox-builtin "IO.removeDirectory.impl.v3")
+(sandbox-builtin "IO.renameDirectory.impl.v3")
+(sandbox-builtin "IO.directoryContents.impl.v3")
+(sandbox-builtin "IO.removeFile.impl.v3")
+(sandbox-builtin "IO.renameFile.impl.v3")
+(sandbox-builtin "IO.getFileTimestamp.impl.v3")
+(sandbox-builtin "IO.getFileSize.impl.v3")
+(sandbox-builtin "IO.serverSocket.impl.v3")
+(sandbox-builtin "Socket.toText")
+(sandbox-builtin "Handle.toText")
+(sandbox-builtin "ThreadId.toText")
+(sandbox-builtin "IO.socketPort.impl.v3")
+(sandbox-builtin "IO.listen.impl.v3")
+(sandbox-builtin "IO.clientSocket.impl.v3")
+(sandbox-builtin "IO.closeSocket.impl.v3")
+(sandbox-builtin "IO.socketAccept.impl.v3")
+(sandbox-builtin "IO.socketSend.impl.v3")
+(sandbox-builtin "IO.socketReceive.impl.v3")
+(sandbox-builtin "IO.kill.impl.v3")
+(sandbox-builtin "IO.delay.impl.v3")
+(sandbox-builtin "IO.stdHandle")
+(sandbox-builtin "IO.process.call")
+(sandbox-builtin "IO.process.start")
+(sandbox-builtin "IO.process.kill")
+(sandbox-builtin "IO.process.wait")
+(sandbox-builtin "IO.process.exitCode")
+(sandbox-builtin "MVar.new")
+(sandbox-builtin "MVar.newEmpty.v2")
+(sandbox-builtin "MVar.take.impl.v3")
+(sandbox-builtin "MVar.tryTake")
+(sandbox-builtin "MVar.put.impl.v3")
+(sandbox-builtin "MVar.tryPut.impl.v3")
+(sandbox-builtin "MVar.swap.impl.v3")
+(sandbox-builtin "MVar.isEmpty")
+(sandbox-builtin "MVar.read.impl.v3")
+(sandbox-builtin "MVar.tryRead.impl.v3")
+(sandbox-builtin "Tls.ClientConfig.default")
+(sandbox-builtin "Tls.ServerConfig.default")
+(sandbox-builtin "Tls.ClientConfig.certificates.set")
+(sandbox-builtin "Tls.ServerConfig.certificates.set")
+(sandbox-builtin "TVar.new")
+(sandbox-builtin "TVar.read")
+(sandbox-builtin "TVar.write")
+(sandbox-builtin "TVar.newIO")
+(sandbox-builtin "TVar.readIO")
+(sandbox-builtin "TVar.swap")
+(sandbox-builtin "STM.retry")
+(sandbox-builtin "IO.ref")
+(sandbox-builtin "Ref.readForCas")
+(sandbox-builtin "Ref.Ticket.read")
+(sandbox-builtin "Ref.cas")
+(sandbox-builtin "Promise.new")
+(sandbox-builtin "Promise.read")
+(sandbox-builtin "Promise.tryRead")
+(sandbox-builtin "Promise.write")
+(sandbox-builtin "Tls.newClient.impl.v3")
+(sandbox-builtin "Tls.newServer.impl.v3")
+(sandbox-builtin "Tls.handshake.impl.v3")
+(sandbox-builtin "Tls.send.impl.v3")
+(sandbox-builtin "Tls.decodeCert.impl.v3")
+(sandbox-builtin "Tls.encodeCert")
+(sandbox-builtin "Tls.decodePrivateKey")
+(sandbox-builtin "Tls.encodePrivateKey")
+(sandbox-builtin "Tls.receive.impl.v3")
+(sandbox-builtin "Tls.terminate.impl.v3")
+(sandbox-builtin "IO.randomBytes")
+(sandbox-builtin "IO.array")
+(sandbox-builtin "IO.arrayOf")
+(sandbox-builtin "IO.bytearray")
+(sandbox-builtin "IO.bytearrayOf")
```

Comment:
I went ahead and double checked these to make sure they're consistent with what's in `Builtin.hs`.